Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -210,7 +210,8 @@ def handle_file_type_fields(raw_type: str, indicator_obj: Dict[str, Any]) -> Non
     indicator_obj['fields'][raw_type.upper()] = hash_value
 
 
-def build_params_dict(tags: List[str], attribute_type: List[str], limit: int, page: int) -> Dict[str, Any]:
+def build_params_dict(tags: List[str], attribute_type: List[str], limit: int, page: int, from_timestamp: str | None = ''
```

Comment:
How can it be none? if the last_run does not contain the timestamp then we assigning empty string to it (not None)